Sources/AccioKit/Services/CarthageBuilderService.swift

```diff
@@ -31,7 +31,12 @@ final class CarthageBuilderService {
             try bash("ln -f -s '\(requiredFrameworkProduct.symbolsFilePath)' '\(productsTargetDirectoryUrl.path)'")
         }
 
+        // remove Cartfile before carthage build as subdependencies have already been built via Accio
```

Comment:
This isn't proper English, I think. Also `carthage` should probably be written with an uppercase first letter. Additionally, please wrap `carthage build` within backticks (`), otherwise the sentence doesn't make any sense gramatically.

Reply:
You're right that `carthage build` should be in backticks, but this way, it's proper English to my understanding:
- `carthage` must not be capitalized as it refers to the command line command
- `remove X before Y as A has already been built via B` seems valid. I'll still insert the word `command` after `Y`~ `carthage build` to make things more clear.

Reply:
Yes, but without the backticks `Y` is actually not `Y` but `Y build` which is weird. But that's fixed once the backticks are added. :)